client/src/app/site/motions/services/motion-filter-list.service.ts

```diff
@@ -276,7 +276,11 @@ export class MotionFilterListService extends BaseFilterListService<ViewMotion> {
 
                         for (const state of workflow.states) {
                             // get the restriction array, but remove the is_submitter condition, if present
-                            const restrictions = state.restriction.filter(r => r !== 'is_submitter');
+                            const restrictions: Permission | Permission[] = state.restriction
+                                .filter(r => r !== Restriction.motionsIsSubmitter)
+                                // without motionsIsSubmitter restrictions are completely equivalent
+                                // with permissions
+                                .map(r => (r as unknown) as Permission);
```

Comment:
@FinnStutzenstein currently, the operator cannot process restrictions correctly.  At some point, restrictions have to be converted back into permissions the allow the operator to check for them.
If you consider this a problem, create a TODO issue for that.

Reply:
This is not a problem. But this shorter version should work, too:
```
const restrictions  = (state.restriction.filter(r => r !== Restriction.motionsIsSubmitter) as unknown) as Permission[]
```